<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Строковый калькултор</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Загрузки/calculator.png</normaloff>../Загрузки/calculator.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="Input">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>511</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans Condensed</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>0</y>
      <width>511</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите выражение</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="AngleBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>116</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Мера углов</string>
    </property>
    <widget class="QRadioButton" name="Grad_box">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>40</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Градусы</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Radian_box">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Радианы</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="POLIZ_box">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>ПОЛИЗ</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="Answer">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>66</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="ValuesForm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>381</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Vera Sans</family>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="title">
     <string>Введеите переменные:</string>
    </property>
    <widget class="QTableWidget" name="InputTable">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>26</y>
       <width>371</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="POLIZ">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>87</y>
      <width>501</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
